test/e2e/windows/host_process.go

```diff
@@ -154,6 +155,176 @@ var _ = SIGDescribe("[Feature:WindowsHostProcessContainers] [Excluded:WindowsDoc
+			},
+			{
+				[]string{"System32\\cmd.exe", "/ver"}, "c:\\Windows",
+			},
+			{
+				[]string{"System32/cmd.exe", "/ver"}, "c:/Windows",
+			},
+			{
+				[]string{"System32/cmd.exe", "/ver"}, "c:/Windows/",
+			},
+			{
+				[]string{"System32\\cmd.exe", "/ver"}, "c:/Windows",
+			},
+			{
+				[]string{"System32\\cmd.exe", "/ver"}, "c:/Windows\\",
+			},
+			{
+				[]string{"System32\\cmd.exe", "/ver"}, "c:\\Windows/",
+			},
+			{
+				[]string{"%CONTAINER_SANDBOX_MOUNT_POINT%\\bin\\uname.exe", "-o"}, "",
+			},
+			{
+				[]string{"%CONTAINER_SANDBOX_MOUNT_POINT%/bin/uname.exe", "-o"}, "",
+			},
+			{
+				[]string{"%CONTAINER_SANDBOX_MOUNT_POINT%\\bin/uname.exe", "-o"}, "",
+			},
+			{
+				[]string{"bin/uname.exe", "-o"}, "%CONTAINER_SANDBOX_MOUNT_POINT%",
+			},
+
+			{
+				[]string{"bin\\uname.exe", "-o"}, "%CONTAINER_SANDBOX_MOUNT_POINT%",
+			},
+			{
+				[]string{"uname.exe", "-o"}, "%CONTAINER_SANDBOX_MOUNT_POINT%/bin",
+			},
+			{
+				[]string{"uname.exe", "-o"}, "%CONTAINER_SANDBOX_MOUNT_POINT%/bin/",
+			},
+			{
+				[]string{"uname.exe", "-o"}, "%CONTAINER_SANDBOX_MOUNT_POINT%\\bin\\",
+			},
+			{
+				[]string{"powershell", "-c", "cmd.exe", "/ver"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "c:/Windows/System32/cmd.exe", "/ver"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "/Windows/System32/cmd.exe", "/ver"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "c:\\Windows\\System32/cmd.exe", "/ver"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "%CONTAINER_SANDBOX_MOUNT_POINT%\\bin\\uname.exe", "-o"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "$env:CONTAINER_SANDBOX_MOUNT_POINT\\bin\\uname.exe", "-o"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "%CONTAINER_SANDBOX_MOUNT_POINT%/bin/uname.exe", "-o"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "$env:CONTAINER_SANDBOX_MOUNT_POINT/bin/uname.exe", "-o"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "bin/uname.exe", "-o"}, "%CONTAINER_SANDBOX_MOUNT_POINT%",
+			},
+			{
+				[]string{"powershell", "-c", "bin/uname.exe", "-o"}, "$env:CONTAINER_SANDBOX_MOUNT_POINT",
+			},
+			{
+				[]string{"powershell", "-c", "bin\\uname.exe", "-o"}, "$env:CONTAINER_SANDBOX_MOUNT_POINT",
+			},
+			{
+				[]string{"powershell", "-c", ".\\uname.exe", "-o"}, "%CONTAINER_SANDBOX_MOUNT_POINT%/bin",
+			},
+			{
+				[]string{"powershell", "-c", "./uname.exe", "-o"}, "%CONTAINER_SANDBOX_MOUNT_POINT%/bin",
+			},
+			{
+				[]string{"powershell", "-c", "./uname.exe", "-o"}, "$env:CONTAINER_SANDBOX_MOUNT_POINT\\bin\\",
+			},
+		}
+
+		// Note: These do not work
+		// []string{"/System32/cmd.exe", "/ver"}, "c:\\Windows\\",
+		// []string{"/System32/cmd.exe", "/ver"}, "c:\\Windows",
+		// []string{".\\bin\\uname.exe", "-o"}, ""
+		// []string{"\\bin\\uname.exe", "-o"}, ""
+		// []string{"/bin/uname.exe", "-o"}, ""
+		// []string{"./bin/uname.exe", "-o"}, ""
+		// []string{"bin\\uname.exe", "-o"}, ""
+		// []string{"bin/uname.exe", "-o"}, ""
```

Comment:
These *should* work if the workdir in the image is C:\ but that's probably not the case